src/mail-app/mail/view/MailViewModel.ts

```diff
@@ -328,6 +325,54 @@ export class MailViewModel {
 		}
 	}
 
+	/*
+		If ConversationInListView is active, all mails in the conversation are returned (so they can be processed in a group)
+		If not, only the primary mail is returned, since that is the one being looked at/interacted with.
+	 */
+	async getActionableMails(mails: Mail[]): Promise<ReadonlyArray<Mail>> {
```

Comment:
if we have it here do we also need it on `ConversationViewModel`? They should be cached anyway